src/components/common/nav-menu/index.js

```diff
@@ -1,25 +1,31 @@
-import React from 'react';
+import React, { PureComponent } from 'react';
 import { Navbar, Nav, NavItem } from 'react-bootstrap';
 import { Link } from 'react-router-dom';
+import { ROUTES } from '../../../constants';
 
-const NavMenu = () => (
-  <div className="nav-menu">
-    <Navbar inverse>
-      <Navbar.Header>
-        <Navbar.Brand>
-          <Link to="/home">Home</Link>
-        </Navbar.Brand>
-      </Navbar.Header>
-      <Nav pullRight>
-        <NavItem>
-          Register
-        </NavItem>
-        <NavItem>
-          Login
-        </NavItem>
-      </Nav>
-    </Navbar>
-  </div>
-);
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
I would just disable this rule in your `eslintconfig`

Reply:
Yup... agreed. I don't really agree with this rule so it might be just worth it to disable it completely.